app/views/layouts/application.html.erb

```diff
@@ -8,9 +8,13 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
```

Comment:
there are nicer ways to include bootstrap in a rails project. This would will force users to download bootstrap from bootstraps CDN, however if we use [the ruby gem](https://github.com/twbs/bootstrap-rubygem) it will be included in our own CSS and JS bundles.